src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7862,8 +7865,8 @@ namespace System.Text.Unicode
 {
     public static partial class Utf8
     {
-        public static System.Buffers.OperationStatus FromUtf16(ReadOnlySpan<char> source, Span<byte> destination, out int charsRead, out int bytesWritten, bool replaceInvalidSequences = true, bool isFinalBlock = true) { throw null; }
-        public static System.Buffers.OperationStatus ToUtf16(ReadOnlySpan<byte> source, Span<char> destination, out int bytesRead, out int charsWritten, bool replaceInvalidSequences = true, bool isFinalBlock = true) { throw null; }
+        public static System.Buffers.OperationStatus FromUtf16(System.ReadOnlySpan<char> source, System.Span<byte> destination, out int charsRead, out int bytesWritten, bool replaceInvalidSequences = true, bool isFinalBlock = true) { throw null; }
+        public static System.Buffers.OperationStatus ToUtf16(System.ReadOnlySpan<byte> source, System.Span<char> destination, out int numBytesRead, out int numCharsWritten, bool replaceInvalidSequences = true, bool isFinalBlock = true) { throw null; }
```

Comment:
I think this needs to remain `bytesRead`, `charsWritten`, and we should modify the implementation instead. Let's make sure we are consistent.

cc @GrabYourPitchforks

Reply:
Looks like this change is already on its way:
https://github.com/dotnet/coreclr/pull/24617
https://github.com/dotnet/corefx/pull/37741

Reply:
Cool, I'll then revert this change. Thanks!